### PR TITLE
[ENT-1395]: Create a Candlepin config to allow external access to a katello events queue

### PR DESCRIPTION
**Pull request by Januson**

### CODE REVIEW

**Comment by barnabycourt** [comment]
Would it be possible to show how to set up a configuration for connecting via a secure connection in addition to an insecure connection?

**Comment by Januson** [comment]
@barnabycourt Added SSL section.

File: docs/candlepin/connect_to_embedded_artemis.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
+---
+title: Allow Remote Connections to embedded Artemis 
+---
+{% include toc.md %}
+
+# Allow Remote Connections to embedded Artemis
+
+
+Following changes need to be done in broker.xml.
```
Comment (barnabycourt) [inline docs/candlepin/connect_to_embedded_artemis.md]: Can a link be given to information about how to configure candlepin to use an external broker.xml?

**Comment by jturel** [comment]
Does this workflow imply, from the Katello perspective for example, that we need to manage the broker.xml entirely? That concerns me because it contains metadata specific to candlepin's concerns. If that changes, then we need to make sure that our broker.xml template (or whatever mechanism we use) is updated to match any updates to broker.xml in this repo.

It'd be nice to have some control exposed via candlepin.conf directly like we do for AMQP, basically just to configure an acceptor for our purpose

**Comment by barnabycourt** [comment]
@jturel Yes, if satellite will need to customize the broker xml. Unfortunately for artemis, this is something that will have to be done in order to do the queue configuration & creation of the acceptor.

**Comment by Januson** [comment]
@jturel Yes. Unless we want to allow connections to the embedded Artemis by default we have to use external broker.xml.

It seems broker.xml supports XInclude so it might be possible to externalize only acceptors. https://activemq.apache.org/components/artemis/documentation/latest/configuration-index.html

File: docs/candlepin/connect_to_embedded_artemis.md (hunk 1 above)
Comment (Januson) [reply docs/candlepin/connect_to_embedded_artemis.md]: Done.

**Pull request merged**